test/kafo/hook_context_test.rb

```diff
@@ -43,5 +44,16 @@ module Kafo
         assert_equal context.module_enabled?('known_module'), true
       end
     end
+
+    describe "#app_option?" do
+      specify do
+        app = {'known_option' => 'option_arg'}
+        config = Minitest::Mock.new
+        kafo.expect :config, config
+        config.expect :app, app
+        assert_equal context.app_option?('known_option'), true
```

Comment:
This is failing. It looks confusing because the [signature](https://docs.ruby-lang.org/en/2.1.0/MiniTest/Assertions.html#method-i-assert_equal) is `assert_equal(exp, act, msg = nil)`. That's why expected and actual are swapped if you look at https://ci.theforeman.org/job/kafo-pr-test/22/testReport/junit/(root)/Kafo__HookContext___app_option_/Test___Matrix___ruby____2_6___PUPPET_VERSION____5_0____test_0001_anonymous/

Looks like the  `#module_enabled?` test is doing the same thing.